lib/action_kit/client.rb

```diff
@@ -0,0 +1,17 @@
+module ActionKit
```

Comment:
This is functionality that should already be available through the `actionkit_connector` gem utilized by the queue, though it's not exposed on the queue.